infrastructure/code/template/details.html

```diff
@@ -408,21 +678,13 @@ <h2 class="severity-title">{{.IssueTitle}}</h2>
         <span data-tab="ignore-details" id="ignore-details-tab" class="tab-item is-selected ignore-details-tab">
           Ignore Details
         </span>
-        <span data-tab="fix-analysis" id="fix-analysis-tab" class="tab-item fix-analysis-tab">
-          Fix Analysis
-        </span>
-        <span data-tab="vuln-overview" id="vuln-overview-tab" class="tab-item vuln-overview-tab">
-          Vulnerability Overview
-        </span>
-        {{else}}
-        <span data-tab="fix-analysis" id="fix-analysis-tab"
-          class="tab-item is-selected sn-fix-analysis fix-analysis-tab">
+        {{end}}
+        <span data-tab="fix-analysis" id="fix-analysis-tab" class="tab-item {{if not .IsIgnored}}is-selected{{end}} sn-fix-analysis fix-analysis-tab">
```

Comment:
Some refactoring here so we have less code duplication and warnings about using the same id twice